src/openms/source/FORMAT/MzTab.cpp

```diff
@@ -587,6 +587,29 @@ namespace OpenMS
     mz_tab_version.fromCellString(String("1.0.0"));
   }
 
+	// static method remapping the target/decoy column from an opt_ to a standardized column
+	static void remapTargetDecoy_(std::vector<MzTabOptionalColumnEntry> &opt_entries)
+	{
```

Comment:
[Travis](http://cdash.openms.de/testDetails.php?test=56758007&build=311424) code style complaint [here](). Cannot see where the space is missing.
```
/home/travis/build/OpenMS/OpenMS/src/openms/source/FORMAT/MzTab.cpp:592:  Missing space before {  [whitespace/braces] [5]
```

Reply:
should be `std::vector<MzTabOptionalColumnEntry>& opt_entries)`?